governance/xc_admin/packages/xc_admin_frontend/components/tabs/General.tsx

```diff
@@ -36,6 +36,22 @@ import PermissionDepermissionKey from '../PermissionDepermissionKey'
 import { PriceRawConfig } from '../../hooks/usePyth'
 import { Wallet } from '@coral-xyz/anchor/dist/cjs/provider'
 
+const MAX_SIZE_ADD_PRODUCT_INSTRUCTION_DATA = 369
+const MAX_SIZE_UPD_PRODUCT_INSTRUCTION_DATA = 403 // upd product has one account less
```

Comment:
Where do these numbers come from?  Are they just randomly selected?  Is there some calculation that leads to them?

Reply:
added a comment